packages/config/config/devices/0x027a/zen25.json

```diff
@@ -55,54 +54,53 @@
 		},
 		{
 			"#": "3",
-			"label": "Wattage Frequency",
-			"unit": "Seconds",
+			"label": "Power (W) Report Frequency",
```

Comment:
This should be consistent with param 2, either remove the `(W)` or add it to param 2.

Reply:
Done